api/src/main/java/org/openmrs/module/mirebalaismetadata/MirebalaisMetadataActivator.java

```diff
@@ -299,38 +299,45 @@ else if (config.getCountry().equals(ConfigDescriptor.Country.MEXICO)) {
     private void installDrugList(Config config) throws IOException {
 
         if (config.getCountry().equals(ConfigDescriptor.Country.HAITI) || config.getCountry().equals(ConfigDescriptor.Country.LIBERIA)) {
+            installSpecificDrugList("HUM_Drug_List-", DRUG_LIST_VERSION, MirebalaisMetadataProperties.GP_INSTALLED_DRUG_LIST_VERSION);
+        } else if (config.getCountry().equals(ConfigDescriptor.Country.MEXICO)) {
+            installSpecificDrugList("CES_Drug_List-", CES_DRUG_LIST_VERSION, MirebalaisMetadataProperties.GP_INSTALLED_CES_DRUG_LIST_VERSION);
+        }
+    }
 
-            int installedDrugListVersion = getIntegerByGlobalProperty(MirebalaisMetadataProperties.GP_INSTALLED_DRUG_LIST_VERSION, -1);
```

Comment:
The code below should be mostly the same, except that I've variablized the GP name, the CSV file name, and the version.